lamindb_setup/core/_settings_instance.py

```diff
@@ -298,7 +298,7 @@ def schema(self) -> set[str]:
     @property
     def _sqlite_file(self) -> UPath:
         """SQLite file."""
-        filepath = self.storage.root / ".lamindb" / "_lamin.db"
+        filepath = self.storage.root / ".lamindb" / "lamin.db"
```

Comment:
```suggestion
        filepath = self.storage.root / ".lamindb/lamin.db"
```

Reply:
I think what you're suggesting is not going to properly resolve on Windows, is it? That's what the `/` is for

Reply:
No, it is fine on windows.

Reply:
Understood that it also works on Windows and you consider it less efficient to use the operator. Let's go with the string!